hamocc/carchm.F90

```diff
@@ -518,13 +523,17 @@ SUBROUTINE CARCHM(kpie,kpje,kpke,kbnd,                                  &
 
 ! Save pco2 w.r.t. dry air for output
        pco2d(i,j) = cu * 1.e6 / Khd
+       !pCO2 wrt moist air
+       pco2m(i,j) = cu * 1.e6 / Kh
 #ifdef natDIC
        natpco2d(i,j) = natcu * 1.e6 / Khd
 #endif
 
 ! Save product of piston velocity and solubility for output
-       kwco2sol(i,j) = kwco2*Kh*1e-6
-
+       kwco2sol(i,j) = kwco2*Kh*1e-6 !m/s mol/kg/muatm 
+       kwco2d(i,j)   = kwco2 ! m/s (incl. ice fraction!)
```

Comment:
Ok, I read the `pco2d` formerly as 2D (not dry) as opposed to `pco2` (as local variable) - so extended it to `kwco2d`, since a local variable `kwco2` is already provided ....